SolarSharp.Interpreter/Security/FunctionBinding/ContextualFunctionRegistry.cs

```diff
@@ -113,7 +113,7 @@ public void MarkContextAsPure(LuaExecutionContext context)
             var contextKey = GetContextKey(context);
             if (!_contextViews.ContainsKey(contextKey))
             {
-                _pureContextCache.AddOrUpdate(context, new object());
+                _pureContextCache.GetOrCreateValue(context);
```

Comment:
Same thing, we don't every mutate this so we can just use GetOrCreateValue.

AddOrUpdate only exists in net8+.